index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen capture -->
 
+<section>
+<h2>Print</h2>
+
+<p>The print functions are a mechanism to render the document to a
+ paginated format. It is returned to the <a>local end</a> as a Base64
+ encoded string containing a PDF representation of the paginated
+ document.
+
+<p class=note>
+For historical reasons, all dimensions relating to the printed output
+are specified in inches.
+
+<p>When required to <dfn>parse a page range</dfn> with
+arguments <var>pageRanges</var> and <var>totalPages</var>, an
+implementation must:
+
+<ol>
+  <li>Let <var>pages</var> be an empty <a>Set</a>
```

Comment:
Surely a list? Order matters, and I can imagine identical pages being present in the output.

Reply:
The way this is used is just to select every page that's in the set, not to change the order. Do print subsystems actually allow you to do things like `3,4,1-2` to change the output order? If they do maybe we ned to change the wording here.

Reply:
I tested and at least Firefox works the way I assumed; you get the pages in the default order but only if they appear in the specified ranges.